providers/openstack/README.md

```diff
@@ -94,43 +101,66 @@ clouds:
 ```
 
 ```bash
-helm upgrade -i csp-helper-my-tenant -n my-tenant --create-namespace https://github.com/SovereignCloudStack/openstack-csp-helper/releases/download/v0.3.0/v0.3.0.tgz -f path/to/clouds.yaml
+CSP_HELPER_VERSION="$(curl https://api.github.com/repos/SovereignCloudStack/openstack-csp-helper/releases/latest -s | jq .name -r)"
+helm upgrade -i csp-helper-"${CS_NAMESPACE}" -n "${CS_NAMESPACE}" --create-namespace https://github.com/SovereignCloudStack/openstack-csp-helper/releases/download/"${CSP_HELPER_VERSION}"/openstack-csp-helper.tgz -f path/to/clouds.yaml
```

Comment:
I would suggest just to use the latest version available with:

`https://github.com/SovereignCloudStack/openstack-csp-helper/releases/latest/download/openstack-csp-helper.tgz`

Reply:
Are you sure there is  always a latest tag?

```
❱ LANG=C wget 'https://github.com/SovereignCloudStack/openstack-csp-helper/releases/download/latest/openstack-csp-helper.tgz'
--2024-06-13 16:49:28--  https://github.com/SovereignCloudStack/openstack-csp-helper/releases/download/latest/openstack-csp-helper.tgz
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-06-13 16:49:28 ERROR 404: Not Found.
```

Reply:
it is pretty nasty, in the link "latest" and "download" swap positions for the latest tag.

Reply:
Adapted all links in document including CSO and CSPO to point to latest.